stdlib/include/builtins.isph

```diff
@@ -73,52 +73,24 @@ EXT READNONE varying float16 __tan_varying_half(varying float16);
 
 // TODO: do they need to be implemented in LLVM IR?
 // AOS/SOA conversion primitives
-EXT inline void __aos_to_soa2_double64(varying double, varying double, NOALIAS varying double *uniform,
```

Comment:
I see these functions in util.m4 but looks like they aren't used anywhere. Is it right? Can they be removed from util.m4 as well?

Reply:
I think that they might be used for targets with width 64. They are removed here because they are purely internal m4 implementations that are not exposed to stdlib.ispc.